.gitignore

```diff
@@ -1,83 +1,28 @@
-.idea/**
-db.json
-# Built application files
-*.apk
-*.ap_
-*.aab
-
-# Files for the ART/Dalvik VM
-*.dex
-
-# Java class files
-*.class
-
-# Generated files
-bin/
-gen/
-out/
-
-# Gradle files
-.gradle/
-build/
-
-# Local configuration file (sdk path, etc)
+.gradle
 local.properties
-
-# Proguard folder generated by Eclipse
-proguard/
-
-# Log Files
+project.properties
+keystore.properties
+*.jks
+uaccount.keystore
+.DS_Store
+*.iml
 *.log
-
-# Android Studio Navigation editor temp files
-.navigation/
-
-# Android Studio captures folder
+*.hprof
+build/
 captures/
-
-# IntelliJ
-*.iml
-.idea/workspace.xml
-.idea/tasks.xml
-.idea/gradle.xml
-.idea/assetWizardSettings.xml
-.idea/dictionaries
-.idea/libraries
-.idea/caches
-# Android Studio 3 in .gitignore file.
-.idea/caches/build_file_checksums.ser
-.idea/modules.xml
-
-# Keystore files
-# Uncomment the following lines if you do not want to check your keystore files in.
-#*.jks
-#*.keystore
-
-# External native build folder generated in Android Studio 2.2 and later
-.externalNativeBuild
-
-# Google Services (e.g. APIs or Firebase)
-# google-services.json
-
-# Freeline
-freeline.py
-freeline/
-freeline_project_description.json
-
-# fastlane
-fastlane/report.xml
-fastlane/Preview.html
-fastlane/screenshots
-fastlane/test_output
-fastlane/readme.md
-
-# Version control
-vcs.xml
-
-# lint
-lint/intermediates/
-lint/generated/
-lint/outputs/
-lint/tmp/
-# lint/reports/
-
+.idea/
+proguard-mappings/
+dexguard-mappings/
+dexguard-license.txt
+dexguard.properties
+DexGuard-8.6.03/*
```

Comment:
you use dexguard?

Reply:
It's generated or I copied it from somewhere.
Either way, it is harmless.